src/components/Modal/Modal.vue

```diff
@@ -57,6 +64,69 @@
 	</script>
 	```
 
+	### Modal with more properties
+
+	```vue
+	<template>
+		<div>
+			<button @click="showModal">Show Modal with fields</button>
+			<modal
+				v-if="modal"
+				@close="closeModal"
+				size="large"
+				title="Title inside modal">
+				<div class="modal__content">
+					<h2>Please enter your name</h2>
+					<TextField label="First Name" :value.sync="firstName" />
+					<TextField label="Last Name" :value.sync="lastName" />
+					<Button
+						:disabled="!this.firstName || !this.lastName"
+						@click="closeModal"
+						type="primary">Submit</Button>
+				</div>
+			</modal>
+		</div>
+	</template>
+	<style scoped>
+	.modal__content {
+		margin: 50px;
+		text-align: center;
+	}
+
+	.input-field {
+		margin: 12px 0px;
+	}
+
+	</style>
+	<script>
+
+	import Button from '../Button/index.js'
```

Comment:
```suggestion
	import ButtonVue from '../Button/index.js'
```
and adjust the use accordingly.

Reply:
Should we actually rename the `Button` component into something else for 6.0.0? Importing like this is a bit painful.

Reply:
> Should we actually rename the `Button` component into something else for 6.0.0? Importing like this is a bit painful.

Fine with me.

Reply:
But this should be done in the context of https://github.com/nextcloud/nextcloud-vue/issues/2643 and shouldn't block this PR here, I think.